image-create

```diff
@@ -207,7 +207,14 @@ try:
         memory_mb = 3072
     else:
         memory_mb = 2048
-    machine = testvm.VirtMachine(verbose=args.verbose, image=args.image, memory_mb=memory_mb, maintain=True)
+
+    console_file = tempfile.NamedTemporaryFile(suffix='.log', prefix='console-')
```

Comment:
This gets changed again in the third commit. Please move all the changes there.

Reply:
But console_file is necessary in second commit. Without it it doesn't work. Or do you mean to squash second and third commit?

Reply:
If they are so interdependent, then just squash them, yes.